mix.exs

```diff
@@ -1,13 +1,33 @@
 defmodule Litmus.MixProject do
   use Mix.Project
 
+  @github_url "https://github.com/shraddha1704/litmus"
```

Comment:
should this be `lob/litmus`?

Reply:
I'm so sorry!. the project was transferred from my account to lob. forgot to make that change!